tests/test_wavefunction.py

```diff
@@ -318,17 +318,9 @@ def circuit(Umat):
             return qml.expval(qml.PauliZ(0))
 
         circuit1 = qml.QNode(circuit, dev)
-        with pytest.raises(ValueError, match="must be a square matrix"):
+        with pytest.raises(ValueError, match="Input unitary must be of shape"):
             circuit1(np.array([[0, 1]]))
 
-        circuit1 = qml.QNode(circuit, dev)
-        with pytest.raises(ValueError, match="must be unitary"):
-            circuit1(np.array([[1, 1], [1, 1]]))
-
-        circuit1 = qml.QNode(circuit, dev)
-        with pytest.raises(ValueError, match=r"must be 2\^Nx2\^N"):
-            circuit1(U)
```

Comment:
Not necessary, but I guess these could be kept and just slightly changed. E.g. to check for the "Operator may not be unitary. Verify unitarity of operation, or use a datatype with increased precision.", and the latter one be another match of "Input unitary must be of shape".